### PR TITLE
DPL - 716,719 - Ability to select v12 revio and v12 well defaults

### LINKED ISSUE #1104: DPL-716 - As a lab user when I create a run I should be able to select SMRT LInk Version 12 for Revio
**User story**
As a lab user when I create a run I should be able to select SMRT LInk Version 12 for Revio

**Who are the primary contacts for this story**
Steve / Craig / James

**Who is the nominated tester for UAT**
Craig / James

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] When I visit the new run page I should see an option for SMRT Link 12 for Revio

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #1104 #1105

Changes proposed in this pull request:

* Reworked pacbio run well default edit to load fields dynamically

### CODE REVIEW

File: src/store/traction/pacbio/runCreate/run.js [hunk 1]
```diff
@@ -44,9 +44,166 @@ const defaultWellAttributes = () => {
...
     demultiplex_barcodes: onInstrument,
     include_fivemc_calls_in_cpg_motifs: 'Yes',
+    movie_acquisition_time: null,
+    include_base_kinetics: null,
+    library_concentration: null,
+    polymerase_kit: null,
   }
 }
 
+const smrtLinkVersionDefaultComponents = {
```
Comment (BenTopping) [inline src/store/traction/pacbio/runCreate/run.js:54]: Feel like it might be more sensible to have this data in the component its used it instead of here.

**Comment by BenTopping** [comment]
> Code looks file but I have tried running it locally. Options change as expected but movie time options don't appear. Also there is no option to select Revio as a Systyem name which might be an idea to add for this story.

Movie times should be there. I had a typo which meant it showed up as Move time which may be where the confusion is. Also system name is pulled from the service so it may be you dont have 12_revio locally or that you dont have it set as active?

**Comment by stevieing** [comment]
> > Code looks file but I have tried running it locally. Options change as expected but movie time options don't appear. Also there is no option to select Revio as a Systyem name which might be an idea to add for this story.
> 
> @stevieing Movie times should be there. I had a typo which meant it showed up as Move time which may be where the confusion is. Also system name is pulled from the service so it may be you dont have 12_revio locally or that you dont have it set as active?

I was thinking about system name not SMRT Link Version. We have added it to the back end but not the ui.

**Comment by BenTopping** [comment]
Ah my mistake. I have added it.

**Review by stevieing — CHANGES_REQUESTED** [review]
For me movie acquisition time still does not have the relevant options? Maybe I am missing something. Also wondering if Include Base Kinetics needs a default but that could wait until UAT.

**Review by stevieing — APPROVED** [review]

**Pull request merged**